tests/app/db.py

```diff
@@ -77,18 +76,84 @@ def create_service(
         restricted=restricted,
         email_from=email_from if email_from else service_name.lower().replace(' ', '.'),
         created_by=user or create_user(email='{}@digital.cabinet-office.gov.uk'.format(uuid.uuid4())),
-        sms_sender=sms_sender,
         prefix_sms=prefix_sms,
     )
 
     dao_create_service(service, service.created_by, service_id, service_permissions=service_permissions)
 
-    if do_create_inbound_number and INBOUND_SMS_TYPE in service_permissions:
-        create_inbound_number(number=sms_sender, service_id=service.id)
+    service.active = active
+    service.research_mode = research_mode
+
+    return service
+
+
+def create_service_with_inbound_number(
```

Comment:
can you make these functions have *args and **kwargs that they pass through to `create_service` to reduce code duplication? eg:


```
def create_service_with_inbound_number(inbound_number='123456', *args, **kwargs):
    service = create_service(*args, **kwargs)
    create_inbound_number(inbound_number, service.id)
    [...]
    return service
```